<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="contrastOrigGray">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gråtone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="garden">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Garden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ocean">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ocean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stillife">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stillife</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contrastColour">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Forsterk</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="constant">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>3.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="contrastOriginal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Kontrastforsterkning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mountains">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mountains</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kontrasten i bildet blir økt. &lt;br/&gt;Dette gjøres ved hjelp av &lt;br/&gt;poisson-likningen&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Velg mellom de ulike bildene&lt;br/&gt;til høyre. Se fargebilde eller &lt;br/&gt;gråtonebilde, eller øk kontrasten &lt;br/&gt;på farge- eller gråtonebildet &lt;br/&gt;ved hjelp av knappene under. &lt;br/&gt;Ved å sette verdien høyere vil&lt;br/&gt;konstrasten økes mer.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balls">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adjuster">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adjuster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contrastCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fog">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trees">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contrastGray">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Forsterk (grå)</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrastImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>76</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Adjuster/Adjuster_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
